README.md

```diff
@@ -71,38 +73,68 @@ useModal('root', {
 });
 ```
 
-`showCloseButton`
-Optional to choose whether to add a close button in the upper right corner.
-Default value is false.
-
-`renderCloseButton`
-Type: (close: () => void) => React.ReactElement
+`components`
 Optional.
-Can be specified when `showCloseButton` is true, allowing you to override the default close button and implement your own close button.
-Use the following
+This is an option to customize the `ModalWrapper` returned by useModal.
+Use as follows
 
 ```jsx
 useModal('root', {
-  showCloseButton: true,
-  renderCloseButton: (close) => (
-    <button
-      style={{
-        position: 'absolute',
-        left: 0,
-        right: 0,
-        bottom: '20px',
-        margin: '0 auto',
-        width: '100px',
-      }}
-      onClick={close}
-      type="button"
-    >
-      Close
-    </button>
-  ),
+  components: {
+    Modal: ({ title, description, children }) => {
+      return (
+        <div
+          style={{
+            padding: '60px 100px',
+            backgroundColor: '#fff',
+            borderRadius: '10px',
+          }}
+        >
+          {title && <h1>{title}</h1>}
+          {description && <p>{description}</p>}
+          {children}
+        </div>
+      );
+    },
+    Overlay: () => {
+      return (
+        <div
+          style={{
+            position: 'fixed',
+            top: 0,
+            left: 0,
+            bottom: 0,
+            right: 0,
+            backgroundColor: 'rgba(0, 0, 0, 0.5)',
+          }}
+        />
+      );
+    },
+    Wrapper: ({ children }) => {
+      return (
+        <div
+          style={{
+            position: 'fixed',
+            top: 0,
+            left: 0,
+            bottom: 0,
+            right: 0,
+            display: 'flex',
+            justifyContent: 'center',
+            alignItems: 'center',
+            zIndex: 1000,
+          }}
+        >
+          {children}
+        </div>
+      );
+    },
+  },
 });
 ```
 
+Combined with `ModalProvider` (described below), you can specify the default style for all `useModal` in the project.
+
 ## Global Settings
 
 The `ModalProvider` component allows you to apply a common default configuration to all `useModal` hooks.
```

Comment:
Is there no more 'value' in the ModalProvider props on line 143?

Reply:
Thankes!
I missed it.